docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,84 @@
    "id": "840ea662",
```

Comment:
<p>Do you think it is helpful to demonstrate all three?</p><p>(why not demonstrate just one?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>umm it just how it was before, but you are right</p>